src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl regService;
+
+    @BeforeAll
+    static void initialize() {
+        regService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void loginIsAlreadyInStorage_NotOk() {
+        User user1 = new User("Alice", "123456", 19);
+        User user2 = new User("Alice", "654321", 22);
+        assertNotNull(regService.register(user1));
+        assertNull(regService.register(user2));
+    }
+
+    @Test
+    void userAgeIsLessThanMustBe_NotOk() {
+        User user = new User("Alice", "123456", 16);
```

Comment:
Make for names, ages and password what you testing constants

Reply:
Thanks, but I'm not sure what is needed in the test class. If the mentor says, I will do it.